bosh/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.2-alpine
+
+ENV BOSH_CLI_VERSION 2.0.48
```

Comment:
Can we use Bosh CLI 3.0.1 due to https://www.cloudfoundry.org/blog/cve-2018-1231/

Reply:
Job's a good'n